Library/Formula/pla.rb

```diff
@@ -0,0 +1,22 @@
+class Pla < Formula
+  homepage "http://www.arpalert.org/pla.html"
+  url "http://www.arpalert.org/src/pla-0.0.tar.gz"
+  version "0.0"
+  sha1 "246deb5ae5b0d41abe88e82cd7940b1ea7516c44"
+
+  depends_on "cairo"
+
+  def install
+    # 0
+    inreplace "Makefile", "/usr/include/cairo", "#{Formula["cairo"].include}/cairo"
+    # 1
+    #ENV.append "CFLAGS", "-I#{Formula["cairo"].include}"
+    # 2
+    #ENV.append_to_cflags "-I#{Formula["cairo"].include}/cairo"
+    # 3
+    #ENV["CFLAGS"] = "-I#{Formula["cairo"].include}/cairo"
+    system "make"
+    bin.install "pla"
+  end
+
```

Comment:
Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!